core/src/display_object/edit_text.rs

```diff
@@ -1954,6 +1954,11 @@ impl<'gc> TInteractiveObject<'gc> for EditText<'gc> {
         point: Point<Twips>,
         _require_button_mode: bool,
     ) -> Option<InteractiveObject<'gc>> {
+        // Don't do anything if run in an AVM2 context.
+        if self.as_displayobject().movie().is_action_script_3() {
```

Comment:
Will this check ever pass? We should only be able to do an AVM2 -> AVM1 transition through a `LoaderDisplay`, which already checks the child movie type to decide whether to call `mouse_pick_avm1` or `mouse_pick_avm2`. I think this should be an assertion instead

Reply:
Never mind - AVM1 can directly load an AVM2 movie into a MovieClip, which will trigger this code path